src/api/app/models/project.rb

```diff
@@ -209,6 +209,18 @@ def subprojects
     Project.where("name like ?", "#{name}:%")
   end
 
+  def siblingprojects
+    p = parent
+    siblings = []
+    unless p.nil?
+      Project.where("name like ? and name != ?", "#{p.name}:%", name).order(:name).each do |sib|
```

Comment:
There is a syntax error (`unexpected ')'` and it should use double quotation mark). I suppose it should be:
`Project.where('name REGEXP (?) and name != ?', "^#{parent.name}:\w*\z", name).order(:name)`

but that doesn't find any sibling projects